### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (#search-chatInput)

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#8]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Personal Banking & Netbanking Services Online - ICICI Bank](https://www.icicibank.com/)
#### Element path
#search-chatInput
#### Snippet
`<input type="text" id="search-chatInput" class="sf-searchbox" autocomplete="off" placeholder="Search for Products, Services.">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.48 (foreground color: #50718c, background color: #feeed3, font size: 13.5pt (18px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 100.0.4896.127
This accessibility issue was found using Accessibility Insights for Web 2.31.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
